README.md

```diff
@@ -157,7 +157,7 @@ Standard IntelliJ dialog for interactive rebase will appear.
 ### Push
 
 After a rebase, you might want to push the rebased branch to the remote.
-Right-click on the given branch and select `Push (Current) Branch` from a context menu.
+Right-click on the given branch and select `Push...` from a context menu.
```

Comment:
While you're here... pls add a mention that selection between target repos is provided by the standard push dialog